src/azure-cli/azure/cli/command_modules/redis/_params.py

```diff
@@ -41,7 +41,8 @@ def load_arguments(self, _):
         c.argument('minimum_tls_version', help='Specifies the TLS version required by clients to connect to cache', arg_type=get_enum_type(TlsVersion))
         c.argument('vm_size', arg_type=get_enum_type(allowed_c_family_sizes + allowed_p_family_sizes), help='Size of Redis cache to deploy. Basic and Standard Cache sizes start with C. Premium Cache sizes start with P')
         c.argument('enable_non_ssl_port', action='store_true', help='If specified, then the non-ssl redis server port (6379) will be enabled.')
-        c.argument('replicas_per_master', help='The number of replicas to be created per master.', is_preview=True)
+        c.argument('replicas_per_master', help='The number of replicas to be created per master.')
+        c.argument('redis_version', help='Redis version. Only major version will be used in create/update request with current valid values: (4, 6)')
```

Comment:
Can you add validator here to check if the given value is 4 or 6?

Reply:
Will prefer to do this validation in resource provider.